pkg/webui/console/containers/webhook-add/webhook-add.js

```diff
@@ -0,0 +1,192 @@
+// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { Component } from 'react'
+import bind from 'autobind-decorator'
+import urlTemplate from 'url-template'
+
+import WebhookForm from '@console/components/webhook-form'
+import WebhookTemplateForm from '@console/components/webhook-template-form'
+
+import { isNotFoundError } from '@ttn-lw/lib/errors/utils'
+import PropTypes from '@ttn-lw/lib/prop-types'
+
+const pathExpand = (url, fields) =>
+  Boolean(url) && url.path ? { path: urlTemplate.parse(url.path).expand(fields) } : url
+
+export default class WebhookAdd extends Component {
```

Comment:
In a separate PR, consider converting this to a functional component, like the edit container already is.

Reply:
Will do